src/main/java/com/exadel/sandbox/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package com.exadel.sandbox.exception;
+
+public class EntityNotFoundException extends RuntimeException{
```

Comment:
Could you please apply Reformat code(optimize imports, cleanup code) for this class